FreeAPS/Sources/Modules/NightscoutConfig/NightscoutConfigStateModel.swift

```diff
@@ -220,9 +238,14 @@ extension NightscoutConfig {
 
                         let targets = fetchedProfile.target_low
                             .map { target -> BGTargetEntry in
-                                BGTargetEntry(
-                                    low: target.value,
-                                    high: target.value,
+                                let median = loop! ? self.getMedianTarget(
+                                    lowTargetValue: target.value,
+                                    lowTargetTime: target.time,
+                                    highTarget: fetchedProfile.target_high, units: self.units
```

Comment:
Apologies for being picky - can we put units on it's own line to be consistent?

Reply:
Thanks for the review. Done.

Reply:
Resolving this conversation as it was addressed with commit `8b8645a`